cobc/field.c

```diff
@@ -2891,6 +2891,11 @@ static int
 validate_field_value (struct cb_field *f)
 {
 	if (f->values) {
+		if (f->flag_picture_l) {
+			cb_error_x (CB_TREE (f),
+				    _("field with PICTURE L may not have "
+				      "a VALUE clause"));
+		}
```

Comment:
I'd suggest to set `f->values = NULL` in this case and have the validation done in the else part.
In any case: `("%s and %s are mutually exclusive"), "PICTURE L", "VALUE")`.